_posts/2017-09-12-repeatable-serverless-applications.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Repeatable Serverless Applications"
+date: 2017-09-11
+categories: serverless architecture docker aws
+author: Rick Foxcroft
+image: img/no_servers.png
+---
+
+Servers can be expensive right? We aren't the biggest fan of lengthy provisioning processes, managing, maintenance and the costly nature that comes with deploying applications on servers. This doesn't mean that we avoid them altogether. We still have a monolithic application that uses several ec2 instances for numerous environments. On one of our recent mob-programming sessions, we decided to tackle the topic of, "How can we turn off any lurking, superfluous and idle ec2 instances" or any other idle instances we may have forgotten to stop/terminate. 
```

Comment:
As an intro you're 'burying the lead' - you talk about serverless and EC2 instances and it's not till we get further down do we find out what the story is about.

I think you need to insert a sentence after the first one which telegraphs the content - i.e. "Building and testing serverless applications locally is hard, but there are tools to help."

And then split out the sentence after that into a new paragraph where you explain we still have EC2 instances which leads to superfluous instances running.